CONTRIBUTING.md

```diff
@@ -0,0 +1,168 @@
+# Contributing to Aligent's CDK Constructs
+
+🙌 Welcome to the Aligent CDK Constructs repository! 🙌    
+
+We appreciate your interest in contributing to this project. Please take a moment to review the guidelines below to ensure a smooth and collaborative contribution process.
+
+By participating in this project, you are expected to adhere to the [Code of Conduct].
+We strive to maintain a welcoming and inclusive environment for all contributors.
+
+_These guidelines are intended to provide direction, but they are not meant to be rigid rules. We trust your judgment 
+and invite you to propose any changes to this document by submitting a pull request. Your feedback is valuable, 
+and we welcome collaborative improvements to these guidelines._
+
+## Table of Contents
+- [Overview](#overview)
+- [How Can I Contribute?](#how-can-i-contribute) 
+- [Getting Started](#getting-started)
+  - [Forking the Repository](#forking-the-repository)
+  - [Setting Up the Development Environment](#setting-up-the-development-environment)
+- [Making Changes](#making-changes)
+  - [Branching Strategy](#branching-strategy)
+  - [Commit Guidelines](#commit-guidelines)
+  - [Pull Request Guidelines](#pull-request-guidelines)
+  - [Updating Documentation](#updating-documentation)
+- [Creating a Release](#creating-a-release)
+  - [Release Tag Format](#release-tag-format)
+
+## Code of Conduct
+
+We are committed to providing a welcoming and inclusive environment for everyone who participates in this project. 
+Please review our [Code of Conduct] to understand the expectations and guidelines for behavior within the community.
+
+## Overview
+This repository contains Aligent's [AWS CDK Constructs].
+
+Below is a list of available constructs:
+
+- [basic-auth](packages/basic-auth) 
```

Comment:
Should this just link to the README rather than listing the constructs? I can see myself forgetting to update this :sweat_smile: